src/js/AuditRule.js

```diff
@@ -47,13 +48,26 @@ axs.AuditRule = function(spec) {
     /**
      * Is this element relevant to this audit?
      * @param {Element} element A potential audit candidate.
+     * @param {Object} flags
```

Comment:
It will always be there, I think it isn't optional? It will be easier to see when I give you the smaller PR.